### ISSUE TITLE
Dynamic content?

### ISSUE THREAD

**Issue by richcorbs** [marmelab/json-graphql-server#91]
I've searched through issues and pull requests and haven't seen this question yet.

Is there a way for json-graphql-server to handle dynamic data passed in from the user?

My idea is to have the user input the JSON which thereafter becomes queryable via graphql.

**Comment by fzaninotto** [comment]
It's not built in, but you can somehow create such a service using the code of this library. 

It's outside of the scope of this project though, so if you want to do it, I invite you to create a new project or to fork this one!

**Comment by richcorbs** [comment]
Hi, I'm back. Thanks for this great software.

I'm trying to do this in Express:

```
app.use('/graphql', getData, jsonGraphqlExpress(data))
```

My getData middleware dynamically retrieves the JSON I'm interested in serving for the particular request. It is called as expected with every request and is successful at retrieving the JSON and setting the value of `data` to the JSON.

The challenge, it seems, is that `jsonGraphqlExpress(data)` initializes the schema on start up with whatever populates `data` at the time and not with every request.

I've spent several hours trying this and that.

I've tried wrapping the call to jsonGraphqlExpress in other middleware and was not successful.

I've tried using schemaBuilder with graphqlHTTP.

I appreciate any tips and suggestions you might have.
